tests/integration/test_interface.py

```diff
@@ -109,7 +109,7 @@ def test_generate_dataset_from_sample_and_source(
             check_original.loc[compare_dataset_idx, col].values
             != check_noised_dataset.loc[compare_dataset_idx, col].values
         ).mean()
-        assert np.isclose(noise_level_dataset, noise_level_sample, rtol=0.07)
+        assert np.isclose(noise_level_dataset, noise_level_sample, rtol=0.08)
```

Comment:
Middle initial causes expected noise to be slightly more off than we anticipate due to all characters not being eligible for OCR noising.  The rtol was ~0.078.

Reply:
well, the relative difference was ~0.078, the tolerance you're allowing for is 0.08.

Reply:
Yes.